programming/sr/vision/index.md

```diff
@@ -146,20 +146,33 @@ orientation
 :   An [`Orientation`](#Orientation) instance describing the orientation of the marker.
 
 spherical
-:   A [`Spherical`](#Spherical) instance describing the position relative to the camera.
+:   A [`SphericalCoordinate`](#SphericalCoordinate) instance describing the position relative to the camera.
 
 cartesian
-:   A [`ThreeDCoordinate`](#ThreeDCoordinate) instance describing the absolute position of the marker relative to the camera.
+:   A [`CartesianCoordinates`](#CartesianCoordinates) instance describing the absolute position of the marker relative to the camera.
 
-[`Coordinate`](#Coordinate) {#Coordinate}
+<a id="Coordinate"/>
+
+[`PixelCoordinates`](#PixelCoordinates) {#PixelCoordinates}
 ---------
 
-A `Coordinate` object contains an `x` and `y` attribute. The exact meaning and unit of these attributes depends on its source.
+A named tuple of `x` and `y` coordinates for the point, in pixels relative to the top left of the image.
+
+~~~~~ python
+print(marker.pixel_centre.x, marker.pixel_centre.y)
+~~~~~
```

Comment:
Can we add some context around what this code block does.

Reply:
7e1b39f